packages/core/test/unit/html/includePanelProcessor.test.ts

```diff
@@ -87,6 +87,44 @@ test('includeFile replaces <include src="doesNotExist.md" optional> with empty <
   expect(result).toEqual(expected);
 });
 
+test('includeFile import footnote from hash', async () => {
+  const indexPath = path.resolve('index.md');
+
+  const index = [
+    '<include src="include.md#exists"/>',
+    '',
+  ].join('\n');
+
+  const include = [
+    '<div id="exists">',
+    '',
+    'text^[footnote]',
+    '</div>',
+    '',
+  ].join('\n');
+
+  const json = {
+    'index.md': index,
+    'include.md': include,
+  };
+
+  fs.vol.fromJSON(json, '');
+
+  const nodeProcessor = getNewDefaultNodeProcessor();
+  const result = await nodeProcessor.process(indexPath, index);
+  // to make lint happy; if write as array.join, some lines more than 100 characters;
```

Comment:
This is really quite hard to read, especially with the necessary spaces... I also don't love the defining of throwaway constants

Can you explore 1) one long string with string concatenation at reasonable points 2) disabling lint for just that section to see if it is more readable?

Reply:
Thanks for the comment; how do you find it after this [commit](https://github.com/MarkBind/markbind/pull/2441/commits/eecebc2f249dcc2afe67c5fe18110a57d6d610cd)?

Reply:
have replied